### ISSUE TITLE
Add Wine-GE to setup.sh installer?

### ISSUE THREAD

**Issue by nerdydumbykier** [Pipetto-crypto/3in1ProotX86Emus#14]
It looks like there are only Wine `8.0` and `8.14` to choose in the install setup. Could it be possible to add Wine-GE `8-13` to the installer?

Thanks.

**Comment by Pipetto-crypto** [comment]
Wine GE is installable from the Start Menu:

Wine->Switch->ge8-13

I won't add it to the installer, at least for now since it's much slower than other options.

**Comment by aaa-sys-potato** [comment]
> Wine GE is installable from the Start Menu:
> 
> Wine->Switch->ge8-13
> 
> I won't add it to the installer, at least for now since it's much slower than other options.

Alright..can you add option to download any wine ..?🙂

**Comment by Pipetto-crypto** [comment]
> > Wine GE is installable from the Start Menu:
> > Wine->Switch->ge8-13
> > I won't add it to the installer, at least for now since it's much slower than other options.
> 
> Alright..can you add option to download any wine ..?🙂

Nope, since every wine need to be recompiled with modifications to work in native

**Comment by aaa-sys-potato** [comment]
> > > Wine GE is installable from the Start Menu:
> > > Wine->Switch->ge8-13
> > > I won't add it to the installer, at least for now since it's much slower than other options.
> > 
> > 
> > Alright..can you add option to download any wine ..?🙂
> 
> Nope, since every wine need to be recompiled with modifications to work in native

I fell you.. okay so can you add another Wine like Wine 8.2 vanilla or staging doesn't matter because this Wine is the best for me and for others..I hope that adding it will be easy for you. If it is difficult it does not matter don't not add it
